backends/core/workers.py

```diff
@@ -949,3 +950,38 @@ def _execute(self):
     job = client.run_async_query(job_name, self._params['query'])
     job.use_legacy_sql = False
     self._begin_and_wait(job)
+
+class AutoMLPredictor(BQWorker):
+  """Worker to run BQML SQL queries in BigQuery."""
```

Comment:
"""Worker to run batch prediction from an AutoML trained model."""

Something like this?